recipes/orange3/meta.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: orange3
+  version: 3.3.6
+
+source:
+  fn: Orange3-3.3.6.tar.gz
+  url: https://github.com/biolab/orange3/archive/3.3.6.tar.gz
+  md5: 927f3702ea5c9f1c8d1b59478476b4d5
+
+build:
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - orange-canvas = Orange.canvas.__main__:main
+  osx_is_app: True
+  skip: True        # [py2k]
+
+requirements:
+  build:
+    - python
```

Comment:
I have added the toolchain requirement

cython however should not be needed for build, as the sources also contain c sources produced by cython